client/src/components/Nav.jsx

```diff
@@ -43,11 +51,11 @@ const Nav = ({ showSidebar, setShowSidebar }) => {
       <NavContainer>
         <CustomFaBars onClick={() => SideBarHandler(showSidebar)} />
         <Profile src={Profilephoto} />
-        <Today>Today</Today>
+        <Today>{title}</Today>
         <OutDropDown>
           <WrapDropDownWrapper>
             <DropDownWrapper>
-              <UserName>Jane Doe</UserName>
+              <UserName>{userInfo.username}</UserName>
```

Comment:
The username require extra refresh to show. I am not sure why, maybe because of api fetching. However, I highly encouraging use the `Auth` instead of passing this props

Reply:
This should be resolved